.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+_Issue #, if available:_
+
+_Description of changes:_
+
+##### Checklist
+
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] This is a code generator change and I have run `yarn run generate-clients -m models`
```

Comment:
This step takes lot of time, and it's not expected for devs to check this box.

How about we add CI to check if generated code is valid TypeScript if there are any code changes in codegen folder?